test-suite-groovy/src/test/groovy/io/micronaut/docs/http/server/netty/websocket/SimpleTextWebSocketSpec.groovy

```diff
@@ -22,6 +22,7 @@ class SimpleTextWebSocketSpec extends Specification {
         when: "a websocket connection is established"
         RxWebSocketClient wsClient = embeddedServer.applicationContext.createBean(RxWebSocketClient, embeddedServer.getURI())
         ChatClientWebSocket fred = wsClient.connect(ChatClientWebSocket, "/chat/stuff/fred").blockingFirst()
+        Thread.sleep(50)
```

Comment:
This seems prone to race conditions and the test will likely fail at random in the future.

Reply:
It already was failing so this is not introducing any new issues